File: Domain/Game.cs
namespace Domain;

public class Game
{
    public int Id { get; set; }
    public bool result { get; set; }
    public virtual Aspect? Aspect { get; set; }
    public int AspectId { get; set; }
    public virtual Hero? Hero { get; set; }
    public int HeroId { get; set; }
    public virtual Difficulty? Difficulty { get; set; }
    public int DifficultyId { get; set; }
    public virtual Scenario? Scenario { get; set; }
    public int ScenarioId { get; set; }
    public bool SchemeEmpty { get; set; }
    public bool SecondarySchemeEmpty { get; set; }
    public bool NotMinions { get; set; }
}

File: Angular/src/app/game.ts
import { Aspect } from "./aspect";
import { Content } from "./content";
import { Difficulty } from "./difficulty";
import { Hero } from "./hero";
import { Scenario } from "./scenario";

export class Game {
  public id : number;
  public result : boolean;
  public aspectId : number;
  public aspect : Aspect;
  public heroId : number;
  public hero : Hero;
  public difficultyId : number;
  public difficulty : Difficulty;
  public scenarioId : number;
  public scenario : Scenario;
  public schemeEmpty : boolean;
  public secondarySchemeEmpty : boolean;
  public notMinions : boolean;

  constructor() {
    this.id = -1;
    this.result = false;
    this.aspectId = -1;
    this.heroId = -1;
    this.difficultyId = -1;
    this.scenarioId = -1;
    this.schemeEmpty = false;
    this.secondarySchemeEmpty = false;
    this.notMinions = false;
    this.aspect = new Aspect();
    this.scenario = new Scenario();
    this.hero = new Hero();
    this.difficulty = new Difficulty();
  }
}
